pydantic/config.py

```diff
@@ -22,7 +24,12 @@
     Callable[[Dict[str, Any], Type[Any]], None],
 ]
 
-Extra = _Extra()
+with warnings.catch_warnings():
+    # The relevant deprecation warning is still raised on attribute access,
+    # so it is okay to suppress the one coming from the @deprecated decorator here.
+    warnings.filterwarnings('ignore', category=PydanticDeprecatedSince20)
+
+    Extra = _Extra()
```

Comment:
Why is it `Extra = _Extra()` and not `Extra = _Extra`?

Reply:
I've completely removed the instantiation of `Extra` now so that it is only re-exported same as `BaseConfig`

Reply:
`getattr(Extra, attribute)` doesn't seem to work without instantiating it first

Reply:
I've reverted the change

Reply:
@Kludex I _think_ we can make that work but then we need to override getattribute on the metaclass. Might help to go over it live so we can confirm it's working with display etc.

Reply:
@dmontagu @Kludex is there anything else I should do or are you taking this over?

Reply:
I'm taking over. Thanks @disrupted 🙏